tests/modules/sitemap/test-sitemap.php

```diff
@@ -106,6 +117,50 @@ public function test_exclude_images() {
 			)
 		);
 	}
+
+	/**
+	 * Test whether the URLs generated by the AMP plugin are added to the sitemap or not.
+	 *
+	 * @ticket 37 XML Sitemap - Add sitemap support for AMP pages.
+	 */
+	public function test_amp() {
+		$amp = 'amp/amp.php';
+		$file = dirname( dirname( AIOSEOP_UNIT_TESTING_DIR ) ) . '/';
+		
+		if ( ! file_exists( $file . $amp ) ) {
+			$this->markTestSkipped( 'AMP not installed. Skipping.' );
```

Comment:
The test was marked to skip

Reply:
yes, skipping this only if the AMP plugin is not found.

Reply:
I probably should of tagged this here too #1527